share_extension/ios/extension_teams.js

```diff
@@ -57,14 +57,15 @@ export default class ExtensionTeams extends PureComponent {
             const myMembers = await Client4.getMyTeamMembers();
             const myTeams = [];
 
-            teams.forEach(async (team) => {
+            for (let i = 0; i < teams.length; i++) {
+                const team = teams[i];
                 const belong = myMembers.find((member) => member.team_id === team.id);
                 if (belong) {
                     const channels = await Client4.getMyChannels(team.id);
```

Comment:
Similar question here, but I don't know if we can even batch this one

Reply:
we can't batch these I think, we could later on with the new endpoint needed to upload large files in the iOS share extension add another endpoint to batch these and only get the default channel for each team.